yolo_detector_node:
  ros__parameters:
    model_path: "install/yolo_detector/share/yolo_detector/models"
    model: "yolov5s.onnx"
    class: "classes.txt"
